src/components/feature-board/BoardState.composable.ts

```diff
@@ -1,26 +1,122 @@
 import { BoardsApiFactory } from "@/serverApi/v3";
 import { $axios } from "@/utils/api";
-import { onMounted, ref } from "vue";
+import { onMounted, reactive, ref } from "vue";
```

Comment:
```suggestion
import { onMounted, ref } from "vue";
```
We don't use reactive here, so we can remove it